internal/commands/kubernetes/versions.go

```diff
@@ -0,0 +1,42 @@
+package kubernetes
+
+import (
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/output"
+
+	"github.com/UpCloudLtd/upcloud-go-api/v4/upcloud/request"
+)
+
+// VersionsCommand creates the "kubernetes versions" command
+func VersionsCommand() commands.Command {
+	return &versionsCommand{
+		BaseCommand: commands.New("versions", "List available versions for Kubernetes node groups", "upctl kubernetes versions"),
```

Comment:
Will the version parameter be for cluster or node-group?

Reply:
You are correct! It is indeed the cluster version, will fix this.

Reply:
Fixed